src/routes/image.rs

```diff
@@ -71,6 +71,8 @@ pub enum ImageProcessingError {
     ProcessingWorkerJoinError,
     #[error("the image processing with libvips has failed")]
     LibvipsProcessingFailed(libvips::error::Error),
+    #[error("The image exceeds the allowed size")]
```

Comment:
For the sake of consistency please start the error message with lowercase similarly to the other messages.

Reply:
absolutely 👍